### PR TITLE
[15.0][MIG] stock_landed_costs_purchase_auto: Migration to 15.0

### PR DESCRIPTION
**Pull request by victoralmau**
Migration to 15.0

Please @pedrobaeza and @CarlosRoca13 can you review it?

@Tecnativa TT41587

### CODE REVIEW

File: stock_landed_costs_purchase_auto/models/purchase_order.py [hunk 1]
```diff
@@ -32,7 +32,7 @@ def _create_picking_with_stock_landed_cost(self, picking):
         self.ensure_one()
         landed_cost = (
             self.env["stock.landed.cost"]
-            .with_context(force_company=self.company_id.id)
+            .with_company(self.company_id.id)
```
Comment (pedrobaeza) [inline stock_landed_costs_purchase_auto/models/purchase_order.py]:
```suggestion
            .with_company(self.company_id)
```

**Review by pedrobaeza — APPROVED** [review]

**Review by CarlosRoca13 — APPROVED** [review]

**Comment by victoralmau** [comment]
CI error is related to https://github.com/OCA/odoo-test-helper/pull/22#issuecomment-1423775170

**Comment by victoralmau** [comment]
Now, it's ready to merge.

**Pull request merged**